createlinks

```diff
@@ -38,6 +38,18 @@ event_templates('nethserver-cockpit-update', qw(
     /etc/nethserver/ldappasswd.conf
     /etc/sudoers.d/30_nethserver_cockpit_roles
     /etc/sudoers.d/55_nsapi_perms
+    /etc/nethserver/cockpit.allow
+    /etc/pam.d/cockpit
+));
+
+# expand cockpit PAM following `ShellOverrideStatus`
+event_templates('nethserver-sssd-save', qw(
+    /etc/pam.d/cockpit
```

Comment:
Expanding also the cockpit.allow template does not waste. Use case: 

1. configure a remote account provider with custom `admins/group` prop
2. enable `ShellOverrideStatus`
3. run `nethserver-sssd-save`

```suggestion
    /etc/pam.d/cockpit
    /etc/nethserver/cockpit.allow
```

See also the next comment

Reply:
done